README.md

```diff
@@ -1 +1,3 @@
 # test-pullapprove
+
+foo bar baz
```

Comment:
```suggestion
!foo bar baz
```

Reply:
I have to say, I heartily disagree with this.

Reply:
You're right, I don't know how to rubies 😭